src/Microsoft.Performance.SDK.Runtime/Discovery/AssemblyExtensionDiscovery.cs

```diff
@@ -191,14 +216,16 @@ public void ProcessAssemblies(IEnumerable<string> directoryPaths)
             }
 
             watch.Stop();
-            Console.Error.WriteLine("Loaded all in {0}", watch.Elapsed);
+            Console.Error.WriteLine("Loaded {0} in {1}", allLoaded ? "all" : "some", watch.Elapsed);
```

Comment:
since you're already touching this, would you mind changing watch.Elapsed to watch.ElapsedMilliseconds and then add a 'ms' unit to the string?

Elapsed provides a TimeSpan, and it's output doesn't strike me as particularly intuitive.